gravitee-apim-console-webui/src/management/api/api-traffic-v4/webhook-logs/components/webhook-logs-quick-filters/webhook-logs-quick-filters.component.ts

```diff
@@ -0,0 +1,290 @@
+import { UntypedFormControl, UntypedFormGroup, ReactiveFormsModule } from '@angular/forms';
+import { MatButtonModule } from '@angular/material/button';
+import { MatChipsModule, MatChipInputEvent } from '@angular/material/chips';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatIconModule } from '@angular/material/icon';
+import { MatInputModule } from '@angular/material/input';
+import { MatSelectModule } from '@angular/material/select';
+import { distinctUntilChanged, takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+import { isEqual } from 'lodash';
+
+import { WebhookLogsApplicationsFilterComponent } from './components/webhook-logs-applications-filter/webhook-logs-applications-filter.component';
+
+import { DEFAULT_PERIOD, MultiFilter, PERIODS, SimpleFilter } from '../../../runtime-logs/models';
+import {
+  DEFAULT_WEBHOOK_LOGS_FILTERS,
+  WebhookLogsQuickFilters,
+  WebhookLogsQuickFiltersInitialValues,
+  WebhookMoreFiltersForm,
+} from '../../models/webhook-logs.models';
+import { WebhookLogsMoreFiltersComponent } from '../webhook-logs-more-filters/webhook-logs-more-filters.component';
+
+type QuickFiltersFormValue = {
+  searchTerm: string;
+  statuses: number[];
+  applications: string[];
+  period: SimpleFilter;
+};
+
+const DEFAULT_FORM_VALUE: QuickFiltersFormValue = {
+  searchTerm: '',
+  statuses: [],
+  applications: [],
+  period: DEFAULT_PERIOD,
+};
+
+@Component({
+  selector: 'webhook-logs-quick-filters',
+  templateUrl: './webhook-logs-quick-filters.component.html',
+  styleUrls: ['./webhook-logs-quick-filters.component.scss'],
+  standalone: true,
+  imports: [
+    CommonModule,
+    DatePipe,
+    ReactiveFormsModule,
+    MatButtonModule,
+    MatChipsModule,
+    MatFormFieldModule,
+    MatIconModule,
+    MatInputModule,
+    MatSelectModule,
+    WebhookLogsApplicationsFilterComponent,
+    WebhookLogsMoreFiltersComponent,
+  ],
+})
+export class WebhookLogsQuickFiltersComponent implements OnInit, OnDestroy {
+  private unsubscribe$: Subject<boolean> = new Subject<boolean>();
+  private _loading = false;
+
+  // TODO: Backend Integration Required
+  // - Verify that availableStatuses and availableCallbackUrls are provided from backend API response
+  // - Consider fetching these dynamically from backend if not included in main logs API response
+  // - Update filter options when new statuses or callback URLs are discovered in API responses
+  // - Ensure filter validation matches backend API requirements
+
+  @Input() availableStatuses: number[] = [];
+  @Input() availableCallbackUrls: string[] = [];
+  @Input() initialValues: WebhookLogsQuickFiltersInitialValues;
+  @Input()
+  get loading() {
+    return this._loading;
+  }
+  set loading(value: boolean) {
+    this._loading = value;
+    if (value) {
+      this.quickFiltersForm?.disable({ emitEvent: false });
+    } else {
+      this.quickFiltersForm?.enable({ emitEvent: false });
+    }
+  }
```

Comment:
By using signals, you can control the lifecycle well and start to do some nuanced stuff. Since the "set" wasn't really setting anything, you can have an auxiliary method with "effect" to handle the quickFiltersForm changes.

```suggestion
  loading = input<boolean>(false);
  
  ....
  constructor() {
     effect(() => {
        const isLoading = this.loading();
        
        if (isLoading) {
           this.quickFiltersForm?.disable({ emitEvent: false });
        } else {
           this.quickFiltersForm?.enable({ emitEvent: false });
        }
     })
  }
```

Reply:
Okay cool! Updated with [282bfa9](https://github.com/gravitee-io/gravitee-api-management/pull/14023/commits/282bfa968140bbbf2c5c27a08ac301ba9ad25b30).